### ISSUE TITLE
Pre-tarined model cannot reproduce the expected accuracy.

### ISSUE THREAD

**Issue by JiangPQ** [jiecaoyu/XNOR-Net-PyTorch#17]
Hi! 
I tested the pre-trained model on ImageNet but only get Top1 at 4% and Top5 at 12% which was far away from the asserted accuracy.
I think the gap may partly caused by some different image pre-processing procedures.
Here is my transform:

    normalize = Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])

    train_transform = Compose([
        RandomResizedCrop(224),
        RandomHorizontalFlip(),
        ToTensor(),
        normalize,
    ])
    
    val_transform = Compose([
        Resize(256),
        CenterCrop(224),
        ToTensor(),
        normalize,
    ])
But I think only the difference between pre-processing is not enough to cause such a huge gap.
Do you have any ideas?
Thank you!

**Comment by jiecaoyu** [comment]
Hi @JiangPQ , I think this is because of the data preprocessing. I think I am using data in the range [0, 255] and I only subtract inputs with the mean values.

**Comment by JiangPQ** [comment]
@jiecaoyu 
Got it.
Thank you!
